### ISSUE TITLE
[Infra] Enable E2E Relay Load Testing on DevNet

### ISSUE THREAD

**Issue by Olshansk** [pokt-network/poktroll#136]
## Objective

Set a foundation for load testing relays on DevNet to enable all scalability efforts & estimations in preparation for DevNet

## Origin Document

Load testing on LocalNet is a blocker for this effort:

- #135

## Goals

- Set the foundation for all future scalability and "bring to prod" efforts of the Shannon upgrade
- Enable us to start getting some data on how much load the network can handle and start facing potential issues

## Deliverables

- [x] Select a tool (framework, library, build, etc...) to load test E2E relay on DevNet if they are different from LocalNet
- [x] Provide a trigger (e.g. ArgoCD) to start & end load testing on DevNet
- [x] Document this process

## Non-goals / Non-deliverables

- Randomizing the relay type being sent
- Telemetry & observability into how the node behaves

## General deliverables

- [ ] **Comments**: Add/update TODOs and comments alongside the source code so it is easier to follow.
- [ ] **Testing**: Add new tests (unit and/or E2E) to the test suite.
- [ ] **Makefile**: Add new targets to the Makefile to make the new functionality easier to use.
- [ ] **Documentation**: Update architectural or development READMEs; use [mermaid](https://mermaid-js.github.io/mermaid/) diagrams where appropriate.

---

**Creator**: @Olshansk 
**Co-Owners**: @okdas

**Comment by okdas** [comment]
Currently contingent on #521, we will then prioritize preparing the infrastructure for running tests on TestNet. In parallel, I'll integrate them on DevNets. As discussed, these will not be the DevNets provisioned via GitHub labels but configured in the `protocol-infra` repo. This allows us to set up workloads as desired and pre-provision the infrastructure before running the tests.
